### PR TITLE
add drawing app

### PR DESCRIPTION
**Pull request by tigercoding56**
remember those toys that had a unicorn that you could slide around a ice scating rink mockup which would leave behind a trail -- which would gradually vanish , well this is the same thing  basically  , Oh its just me who remembers that   
DrawApp

### CODE REVIEW

**Review by fgaz — COMMENTED** [review]
Nice one!

I see some extraneous git history, could you squash those commits?

File: apps/draw.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+# Copyright (C) 2020 Daniel Thompson
+"""The complete set of wasp-os application entry points are documented
+below as part of a template application. Note that the template does
+not rely on any specific parent class. This is because applications in
+wasp-os can rely on *duck typing* making a class hierarchy pointless.
+"""
```
[the remark this answers is not in the document]
Comment (tigercoding56) [reply apps/draw.py:7]: ok  ,

File: apps/draw.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+       launcher. Applications that expect to be installed on the quick
+       ring will not be listed by the launcher and need not provide any
+       icon. When no icon is provided the system will use a default
+       icon.
+       The icon is an opportunity to differentiate your application from others
+       so supplying an icon is strongly recommended. The icon, when provided,
+       must not be larger than 96x64.
+    """
+    NAME = 'Draw'
+    ICON = icons.app
```
[the remark this answers is not in the document]
Comment (tigercoding56) [reply apps/draw.py:9]: ok  , im bad at art though 
draw_icon
-- (i

File: apps/draw.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+                    self.cursor = cursor2
+        else:
+                print("#")
+                print(cursor2)
+
+        return False
+
+    def _update(self):
+        """Update the dynamic parts of the application display."""
+        pass
```
[the remark this answers is not in the document]
Comment (tigercoding56) [reply apps/draw.py]: ok

**Comment by tigercoding56** [comment]
> Nice one!
> 
> I see some extraneous git history, could you squash those commits?

I'll try    (i am not good at git )

**Comment by fgaz** [comment]
> i do not know how to squash commits

I see you're using the github web interface to publish your changes. Sadly squashing is not possible from there.

That by itself isn't a problem, we can squash on merge. However, we also need your commits to be [signed off](https://wasp-os.readthedocs.io/en/latest/contributing.html), and that you can only do yourself with git. Maybe join [the project's chat room](https://github.com/wasp-os/wasp-os#community) so we can help you with it

**Comment by tigercoding56** [comment]
> > i do not know how to squash commits
> 
> I see you're using the github web interface to publish your changes. Sadly squashing is not possible from there.
> 
> That by itself isn't a problem, we can squash on merge. However, we also need your commits to be [signed off](https://wasp-os.readthedocs.io/en/latest/contributing.html), and that you can only do yourself with git. Maybe join [the project's chat room](https://github.com/wasp-os/wasp-os#community) so we can help you with it

ok

**Comment by tigercoding56** [comment]
Ok did i sign off the  commits correctly ?   
image

**Comment by tigercoding56** [comment]
ok      it says review needed  so requesting review

**Comment by fgaz** [comment]
Hmm, looks like something went wrong and the drawing app commits got replaced by the flappybird app and some older commits, not sure why. Maybe it'd be easier to start from scratch: you could make a copy of the drawing app file, make a new branch from master and a new pull request.

**Comment by tigercoding56** [comment]
> Hmm, looks like something went wrong and the drawing app commits got replaced by the flappybird app and some older commits, not sure why. Maybe it'd be easier to start from scratch: you could make a copy of the drawing app file, make a new branch from master and a new pull request.

ok

**Comment by tigercoding56** [comment]
> > Hmm, looks like something went wrong and the drawing app commits got replaced by the flappybird app and some older commits, not sure why. Maybe it'd be easier to start from scratch: you could make a copy of the drawing app file, make a new branch from master and a new pull request.
> 
> ok

closing   as  branch  weirdness is happening (reopen later)

**Pull request closed without merging**